pycon/settings.py

```diff
@@ -174,61 +163,44 @@
     'social.pipeline.user.create_user',
     # THIS IS IMPORTANT!!!! Connect new authenticated users to profiles
     # of the important project apps!!
-    'p3.views.profile.connect_profile_to_assopy',
+    'conference.accounts.social_connect_profile',
     'social.pipeline.social_auth.associate_user',
     'social.pipeline.social_auth.load_extra_data',
     'social.pipeline.user.user_details'
 )
 
 TEMPLATES = [
-    # Jinja templates from ep2019+
-    {
-        'BACKEND': 'django.template.backends.jinja2.Jinja2',
```

Comment:
i removed it because atm we will run just with django templates.
we might bring it back when we're going to work on the schedule page, but maybe this will be SPA anyway, will see...
for now the biggest problem when moving to jinja2 was using django CMS with it, so we'd have to default to django templates.
maybe there's a way to make it work with jinja, but I couldn't figure out how to connect the two in any nice way, so I'd have to default back to django templates.